assets/js/phoenix_live_view.js

```diff
@@ -480,7 +480,7 @@ export class LiveSocket {
       let target = closestPhxBinding(e.target, click)
       let phxEvent = target && target.getAttribute(click)
       if(!phxEvent){ return }
-      e.preventDefault()
+      e.stopPropagation()
```

Comment:
We probably need to take a look at this [commit](https://github.com/phoenixframework/phoenix_live_view/commit/1d0038bd272657a35981bbd68150b087003c6d52) and ensure we aren't regressing on something else.  If we are, we can also look to an alternative solution.

Reply:
Looks like this was fixed 💪 Ignore this comment.